rakelib/release.rake

```diff
@@ -134,27 +153,48 @@ namespace :release do
     JSON.parse(response.body)
   end
 
+  def github_upload(json, filename)
+    upload_url = json['upload_url'].gsub(/\{.*\}/, "?name=#{filename}")
+    zipfile = "#{BUILD_DIR}/#{filename}"
```

Comment:
nit (I know, I'm the one who wrote it using string interpolation in the first place 😂  but might as well use the occasion to update it now that I know Ruby a bit better a couple of years later ^^):
```suggestion
    zipfile = File.join(BUILD_DIR, filename)
```